pycbc/inference/models/marginalized_gaussian_noise.py

```diff
@@ -28,7 +28,7 @@
 from .gaussian_noise import (BaseGaussianNoise, create_waveform_generator)
```

Comment:
@yi-fan-wang You forgot to import GaussianNoise when you made this change

Reply:
Fixed